test/downstream/symbol_indexing.jl

```diff
@@ -66,8 +66,34 @@ sol = solve(prob, Rodas4())
 @test sol[α] isa Vector
 @test sol[α, 3] isa Float64
 @test length(sol[α, 5:10]) == 6
-# @test sol[γ] isa Real
-# @test sol[γ] == 2.0
+@test sol[γ] isa Real
+@test sol[γ] == 2.0
+@test sol[(lorenz1.σ, lorenz1.ρ)] isa Tuple
+
+@test sol[[lorenz1.x, lorenz2.x]] isa Vector{Vector{Float64}}
+@test length(sol[[lorenz1.x, lorenz2.x]]) == length(sol)
+@test all(length.(sol[[lorenz1.x, lorenz2.x]]) .== 2)
+@test sol[(lorenz1.x, lorenz2.x)] isa Vector{Tuple{Float64, Float64}}
+@test length(sol[(lorenz1.x, lorenz2.x)]) == length(sol)
+@test all(length.(sol[(lorenz1.x, lorenz2.x)]) .== 2)
+
+@test sol[[lorenz1.x, lorenz2.x], :] isa Matrix{Float64}
+@test size(sol[[lorenz1.x, lorenz2.x], :]) == (2, length(sol))
+@test size(sol[[lorenz1.x, lorenz2.x], :]) == size(sol[[1, 2], :]) == size(sol[1:2, :])
+
+@variables q(t)[1:2] = [1.0, 2.0]
+eqs = [D(q[1]) ~ 2q[1]
+       D(q[2]) ~ 2.0]
+@named sys2 = ODESystem(eqs, t, [q...], [])
+sys2_simplified = structural_simplify(sys2)
+prob2 = ODEProblem(sys2, [], (0.0, 5.0))
+sol2 = solve(prob2, Tsit5())
+
+@test sol2[q] isa Vector{Vector{Float64}}
+@test sol2[(q...,)] isa Vector{NTuple{length(q), Float64}}
```

Comment:
```suggestion
@test_broken sol2[(q...,)] isa Vector{NTuple{length(q), Float64}}
```

Reply:
This isn't broken though

Reply:
Oops? 😅  https://github.com/SciML/SciMLBase.jl/actions/runs/3553162451/jobs/5968519445#step:6:778 Wrong test lol